README.md

```diff
@@ -18,14 +18,17 @@ For general questions about using the virtual environments or writing your Actio
 | Windows Server 2019 | `windows-latest` or `windows-2019` | [windows-2019] | [![](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=windows-2019&badge=1)](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=windows-2019&redirect=1)
 | Windows Server 2016 | `windows-2016` | [windows-2016] | [![](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=windows-2016&badge=1)](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=windows-2016&redirect=1)
 ```
-The Ubuntu 20.04 virtual environment is currently provided as a preview only.
-The "ubuntu-latest" YAML workflow label still uses the Ubuntu 18.04 virtual environment.
-```
-```
 The MacOS 11.0 virtual environment is currently provided as a preview only.
 The "macos-latest" YAML workflow label still uses the MacOS 10.15 virtual environment.
 ```
 
+***What images are available for GitHub Actions and Azure DevOps?***
+The availability of images for GitHub Actions and Azure DevOps is different. See documentation for more details:  
+- [GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners#supported-runners-and-hardware-resources)  
+- [Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#software)  
+
+***What image version is used in my build?*** Usually, image deployment takes 3-4 days and documentation in `main` branch is updated only when deployment is finished. To find out which image version and what software versions are used in specific build, see logs of `Set up job` (GitHub Actions) or `Initialize job` (Azure DevOps) steps.

```

Comment:
see logs of `Set up job` (GitHub Actions) or `Initialize job` (Azure DevOps) steps -> see `Set up job` (GitHub Actions) or `Initialize job` (Azure DevOps) step log